projects/core/src/lib/operators/dialogs/bandwise-expression-operator/bandwise-expression-operator.component.html

```diff
@@ -0,0 +1,65 @@
+<geoengine-sidenav-header>Calculate Expression on Raster</geoengine-sidenav-header>
+
+<form [formGroup]="form" (ngSubmit)="add()" *ngIf="projectHasRasterLayers$ | async; else noDataInfo">
+    <geoengine-operator-dialog-container [loading]="loading$ | async | valueDefault: false">
+        <a mat-icon-button operatorInputIcon href="https://docs.geoengine.io/operators/expression.html" target="_blank">
```

Comment:
Should this link to the bandwise-expression instead?

Reply:
6d7e89a21b3709cb795936875bd451af6d753621